packages/lib-classifier/src/components/Classifier/components/FieldGuide/components/FieldGuide/components/FieldGuideItemAnchor/FieldGuideItemAnchor.js

```diff
@@ -45,11 +41,10 @@ function FieldGuideItemAnchor({
   }, [itemIndex, onClick])
 
   const label = <AnchorLabel icons={icons} item={item} title={title} />
-  const anchorColor = theme.dark ? 'light-3' : 'dark-5'
   return (
     <Anchor
       className={className}
-      color={anchorColor}
+      color={{ light: 'dark-5', dark: 'light-3' }}
```

Comment:
Grommet handles this without needing to handle `anchorColor` manually. Storybook can be used to double check light/dark theme.